src/benchmark/CMakeLists.txt

```diff
@@ -44,9 +44,16 @@ target_link_libraries(
 )
 
 # General purpose benchmark runner
-add_executable(hyriseBenchmark benchmark_runner_main.cpp)
+add_executable(
+    hyriseBenchmarkFiles
+
+    file_based_benchmark.cpp
```

Comment:
"file_based" feels clunky to me. Why not just "generic" benchmark?

Reply:
"generic" doesn't convey any meaning to me.

Reply:
mmh, okay, if you think "file_based" is fine, then so be it